shimmingtoolbox/cli/download_data.py

```diff
@@ -24,6 +23,7 @@ def main(verbose, output, data):
     """
     Download data from the internet.
     """
+    # TODO: logging does not seem to output on the terminal
```

Comment:
Is that possibly because you're only testing with pytest? Pytest eats stdout by default. You have to run it as `pytest -s` to see messages.

Reply:
If that is the reason (and I'm pretty sure it is):

```suggestion
```